meta/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+galaxy_info:
+  author: haslersn
+  description: Installs Jitsi Meet; see https://github.com/jitsi/jitsi-meet
+  company: stuvus (https://stuvus.uni-stuttgart.de)
+
+  license: AGPLv3
```

Comment:
In the past we decided to use the _Creative Commons Attribution Share Alike 4.0 International
_ license.